website/source/docs/providers/template/r/file.html.md

```diff
@@ -14,7 +14,7 @@ Renders a template from a file.
 
 ```
 resource "template_file" "init" {
-    filename = "${path.module}/init.tpl"
+    template = "${file(${path.module}/init.tpl)}"
```

Comment:
@phinze I was updating our configuration due to this PR and tried to update to the configuration above, but it's not proper syntax. It should be `template = "${file("${path.module}/init.tpl")}"`